src/bitwarden_exporter/settings.py

```
@@ -103,17 +104,38 @@ def get_bitwarden_settings_based_on_args() -> BitwardenExportSettings:
+        encoding="utf-8",
+    ) as f:
+        cli_version = f.read()
+
+    parser.add_argument(
+        "--version",
+        action="version",
+        version=cli_version,
+    )
+
+    with open(
+        os.path.join(os.path.dirname(os.path.realpath(__file__)), "resources", "ascii.txt"), "r", encoding="utf-8"
```

Ruff (0.14.1)

Unnecessary mode argument

Remove mode argument

(UP015)